scalyr_agent/monitor_utils/k8s.py

```diff
@@ -710,9 +714,15 @@ class _CacheConfig( object ):
     Internal configuration options for the Kubernetes cache
     """
 
-    def __init__( self, api_url="https://kubernetes.default", verify_api_queries=True, cache_expiry_secs=30,
-                  cache_purge_secs=300, cache_expiry_fuzz_secs=0, cache_start_fuzz_secs=0,
-                  query_timeout=20, log_api_responses_to_disk=False, agent_log_path=None):
+    def __init__(self, api_url="https://kubernetes.default", verify_api_queries=True, cache_expiry_secs=30,
+                 cache_purge_secs=300, cache_expiry_fuzz_secs=0, cache_start_fuzz_secs=0,
+                 query_timeout=20,
+                 log_api_responses=False,
```

Comment:
As an idea for another day, I wonder if it might be worth refactoring all of logging of api calls to files in to a separate class, and then passing a single object down, rather than an extra 5 params.  Not necessary for this commit, but something to think about.